lesson_14/exceptions/exceptions_app/src/main/java/com/codedifferently/lesson14/ecommerce/EcommerceSystem.java

```diff
@@ -17,19 +17,30 @@ public void addProduct(String productId, String name) {
     products.put(productId, new Product(productId, name));
   }
 
-  public String placeOrder(String productId, int quantity) {
+  public String placeOrder(String productId, int quantity)
+      throws ProductNotFoundException, Exception {
```

Comment:
Don't need `Exception` here.

Reply:
Exception removed. It came up because I used 'Quick Fix' somewhere in the code.